torch/_inductor/triton_heuristics.py

```diff
@@ -53,6 +53,45 @@ class HeuristicType(Enum):
     TEMPLATE = auto()
 
 
+class AutotuneHint(Enum):
+    ELEMENTS_PER_WARP_32 = 0
+
+    # Triton codegen tries to codegen set of AutotuneHints.
+    # Enum.__repr__ looks like "<AutotuneHint.ELEMENTS_PER_WARP_32: 0>""
+    # which isn't valid python.
+    # Enum.__str__ will just return "AutotuneHint.ELEMENTS_PER_WARP_32".
+    __repr__ = Enum.__str__
+
+
+def autotune_hints_to_configs(
+    hints: Set[AutotuneHint], size_hints, block_size
+) -> List[Config]:
+    configs = []
+
+    for hint in hints:
+        if hint == AutotuneHint.ELEMENTS_PER_WARP_32:
+            if len(size_hints) == 1:
+                xyz_options = [(block_size // 4,)]
+            elif len(size_hints) == 2:
```

Comment:
dont think `len(size_hints) != 1` is currently exercised in tests

Reply:
any suggestions how to trigger this? I haven't been able to figure out a way to make this run

Reply:
Try a channels last tensor ?

Reply:
Thanks - added a test in test_torchinductor.py for now, lmk if that is sufficient; if not I can try to add a separate test file for pointwise autotuning